Meadowlark-js/packages/meadowlark-utilities/src/Logger.ts

```diff
@@ -104,6 +105,14 @@ export function initializeLogging(): void {
       }),
     ],
   });
+
+  if (Config.get('SAVE_LOG_TO_FILE') === true) {
+    logger.add(
+      new winston.transports.File({
+        filename: join(process.cwd(), 'meadowlark.log'),
```

Comment:
At some point we'll probably get a requirement to modify the name of the file or add a timestamp or something. And I'm happy to entertain those suggestions when we receive them... in the future.

But, there is one thing we need to change: if writing a file, there needs to be better control of _where_ to write it. We don't want the web directory to be writable. In Docker, we would want the default directory to be `/var/log`.